test/automation-aggregator.ts

```diff
@@ -477,36 +506,22 @@ describe('AutomationAggregatorBot', async () => {
             const triggerCounter = await AutomationBotInstance.triggersCounter()
 
             const triggerIds = [Number(triggerCounter) - 1, Number(triggerCounter)]
-            const dataToSupplyRemove = AutomationBotAggregatorInstance.interface.encodeFunctionData(
-                'removeTriggerGroup',
-                [testCdpId, triggerGroupId, triggerIds, false],
-            )
+            const dataToSupplyRemove = AutomationBotAggregatorInstance.interface.encodeFunctionData('removeTriggers', [
+                triggerIds,
+                false,
+            ])
             await expect(ownerProxy.connect(owner).execute(AutomationBotAggregatorInstance.address, dataToSupplyRemove))
                 .to.not.be.reverted
         })
-        it('should fail to remove a not existing trigger group', async () => {
-            const owner = await hardhatUtils.impersonate(ownerProxyUserAddress)
 
-            const triggerCounter = await AutomationBotInstance.triggersCounter()
-
-            const triggerIds = [Number(triggerCounter) - 1, Number(triggerCounter)]
-            const dataToSupplyRemove = AutomationBotAggregatorInstance.interface.encodeFunctionData(
-                'removeTriggerGroup',
-                [testCdpId, triggerGroupId + 1, triggerIds, false],
-            )
-            await expect(ownerProxy.connect(owner).execute(AutomationBotAggregatorInstance.address, dataToSupplyRemove))
-                .to.be.reverted
-        })
         it('should only remove approval if last param set to true - test FALSE', async () => {
```

Comment:
we can remove the two tests below, ase do not approve the bot or aggregator outside of the tx